kitchen.yml

```diff
@@ -6,7 +6,7 @@ provisioner:
   name: chef_solo
   product_name: chef
   enforce_idempotency: true
-  multiple_converge: 2
+  multiple_converge: 3
```

Comment:
What specifically is causing us to need to bump this to 3 globally?

Reply:
oh sorry, i missed that one, thought it was only in the dokken yml. I can test again but what it was is that the debian/ubuntu ones wouldnt be idempotent until the third converge

Reply:
I must have fixed something after that that allowed them to though so then its vestigial now

Reply:
Then I recommend you revert this back to 2.